lLyrics/LocalParser.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
```

Comment:
I wonder, is it still relevant in 2020, after Python-2's demise?

Reply:
Yep, it's useless. I just copied the template from another parser.